TaskBroker/VstsServerTaskBroker.UnitTest/packages.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net451" />
```

Comment:
1 config file for the repo is much better than N config files as dependencies needs to be of same version